detox/src/devices/Device.js

```diff
@@ -42,6 +43,16 @@ class Device {
     await this.client.sendUserNotification(params);
   }
 
+  async setOrientation(orientation) {
+    // orientation is 'landscape' (meaning left side portrait) or 'portrait' (non-reversed)
+    const call = invoke.call(
```

Comment:
The ObjC code looks like this:
```ObjC
[EarlGrey rotateDeviceToOrientation:UIDeviceOrientationLandscapeLeft errorOrNil:nil];
```

JS code pass `UIDeviceOrientation` which is an ENUM, so I guess the easiest way is to pass NSInteger with the position of the desired orientation ENUM.

Looking at `UIDevice.h`, which holds these ENUMS

```ObjC
typedef NS_ENUM(NSInteger, UIDeviceOrientation) {
    UIDeviceOrientationUnknown,
    UIDeviceOrientationPortrait,            // Device oriented vertically, home button on the bottom
    UIDeviceOrientationPortraitUpsideDown,  // Device oriented vertically, home button on the top
    UIDeviceOrientationLandscapeLeft,       // Device oriented horizontally, home button on the right
    UIDeviceOrientationLandscapeRight,      // Device oriented horizontally, home button on the left
    UIDeviceOrientationFaceUp,              // Device oriented flat, face up
    UIDeviceOrientationFaceDown             // Device oriented flat, face down
} __TVOS_PROHIBITED;
```

For `UIDeviceOrientationPortrait` pass pass `invoke.IOS.NSInteger(1)` 
For `UIDeviceOrientationLandscapeLeft` pass `invoke.IOS.NSInteger(3)`

I think this should work ...
Of course, for readability, we'd like to keep this ENUM values in JS as well.

So it would look something like this:
```js
invoke.IOS.EarlGrey(
         'rotateDeviceToOrientation:errorOrNil:',
        invoke.IOS.NSInteger(orientationEnum)
);
```

I hope it'll work :)